turbinia/task_manager.py

```diff
@@ -156,8 +160,8 @@ def setup(self, jobs_denylist=None, jobs_allowlist=None, *args, **kwargs):
       disabled_jobs = list(set(disabled_jobs) - set(jobs_allowlist))
     if disabled_jobs:
       log.info(
-          'Disabling non-allowlisted jobs configured to be disabled in the '
-          'config file: {0:s}'.format(', '.join(disabled_jobs)))
+          f'Disabling non-allowlisted jobs configured to be disabled in '
+          f'the config file: {"".join(disabled_jobs)}')
```

Comment:
Why no separator for the disabled_jobs list here?  Wouldn't we want at least a space?

Reply:
Good catch. Fixed.